papers/michael_lange/michael_lange.rst

```diff
@@ -460,10 +488,17 @@ efficiently define rigorous forward modelling and adjoint operators
 from high-level symbolic definitions also implies that domain
 scientists are able to quickly adjust the numerical method and
 discretisation to the individual problem and hardware architecture
-[Louboutin17a]_. In the following example we will demonstrate the
-generation of forward and adjoint operators for the acoustic wave
-equation to implement the so-called adjoint test. The governing
-equation is defined as
+[Louboutin17a]_.
+
+In the following example we will demonstrate the generation of forward
+and adjoint operators for the acoustic wave equation and verify their
```

Comment:
Split and ref http://library.seg.org/doi/abs/10.1190/1.3238367 for more details on seismic inversion.

Full PDF at https://jean-virieux.obs.ujf-grenoble.fr/IMG/pdf/GPY_2009_VIRIEUX.pdf?